src/components/Comment.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
This import of react isn't used and is a throwback to a time when it was necessary